system/camerad/cameras/camera_server.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
```

Comment:
let's move the camera server files out of cameras/, that dir should probably just be for the image sensor files